ckanext/validation/i18n/ckanext-validation.pot

```diff
@@ -1,56 +1,56 @@
 # Translations template for ckanext-validation.
-# Copyright (C) 2022 ORGANIZATION
+# Copyright (C) 2024 ORGANIZATION
```

Comment:
regenerating the whole pot and updating the po files create lots of these comment changes that will make it harder to merge. If this feature is going to be upstream and these are included in the PR then it's fine, but otherwise I would paste in just the strings we've added so merging changes back to our fork is easier

Reply:
Good point, for now I will just add the strings into our `strings.py` in the Canada plugin.

Reply:
@wardi oh...nevermind actually. These i18n files are ONLY in our fork/branch. Upstream does not implement any i18n